OsmAnd/src/net/osmand/plus/base/SelectionBottomSheet.java

```diff
@@ -274,6 +274,10 @@ public void onDestroy() {
 		private String description;
 		private int iconId;
 		private Object object;
+
+		private String distance;
```

Comment:
SelectableItem should not have specific fields like distance, points count and time.